services/js-poc/build.rs

```diff
@@ -1,134 +1,51 @@
-    esm = [
-        dir "src/runtime/js",
-        "fleek.js",
-        "global.js",
-        "bootstrap.js"
-    ]
-);
-
-struct Permissions {}
-impl TimersPermission for Permissions {
-    fn allow_hrtime(&mut self) -> bool {
-        unreachable!()
-    }
-}
-impl FetchPermissions for Permissions {
-    fn check_net_url(
-        &mut self,
-        _url: &deno_core::url::Url,
-        _api_name: &str,
-    ) -> Result<(), deno_core::error::AnyError> {
-        unreachable!()
-    }
-    fn check_read(
-        &mut self,
-        _p: &std::path::Path,
-        _api_name: &str,
-    ) -> Result<(), deno_core::error::AnyError> {
-        unreachable!()
-    }
-}
-impl WebSocketPermissions for Permissions {
-    fn check_net_url(
-        &mut self,
-        _url: &deno_core::url::Url,
-        _api_name: &str,
-    ) -> Result<(), deno_core::error::AnyError> {
-        unreachable!()
-    }
-}
-impl NetPermissions for Permissions {
-    fn check_net<T: AsRef<str>>(
-        &mut self,
-        _host: &(T, Option<u16>),
-        _api_name: &str,
-    ) -> Result<(), deno_core::error::AnyError> {
-        unreachable!()
-    }
-
-    fn check_read(
-        &mut self,
-        _p: &std::path::Path,
-        _api_name: &str,
-    ) -> Result<(), deno_core::error::AnyError> {
-        unreachable!()
-    }
-
-    fn check_write(
-        &mut self,
-        _p: &std::path::Path,
-        _api_name: &str,
-    ) -> Result<(), deno_core::error::AnyError> {
-        unreachable!()
-    }
-}
-
-#[derive(Deserialize)]
-struct DenoJson {
-    imports: HashMap<String, String>,
-}
-
-#[derive(Deserialize)]
-struct DenoLock {
-    remote: HashMap<String, String>,
-}
-
-fn create_integrity_url(import: &str, hex_sha256: &str) -> String {
-    let deno_hash = hex::decode(hex_sha256).unwrap();
-    let sri = base64::engine::general_purpose::STANDARD.encode(deno_hash);
-    format!("{import}#integrity=sha256-{sri}")
-}
 
 fn main() {
     let extensions = vec![
         deno_webidl::init_ops_and_esm(),
         deno_console::init_ops_and_esm(),
         deno_url::init_ops_and_esm(),
-        deno_web::init_ops_and_esm::<Permissions>(Arc::new(Default::default()), None),
+        deno_web::deno_web::init_ops_and_esm::<Permissions>(Arc::new(Default::default()), None),
         deno_net::init_ops_and_esm::<Permissions>(None, None),
         deno_fetch::init_ops_and_esm::<Permissions>(Default::default()),
         deno_websocket::init_ops_and_esm::<Permissions>(Default::default(), None, None),
         deno_crypto::init_ops_and_esm(None),
         deno_webgpu::init_ops_and_esm(),
         deno_canvas::init_ops_and_esm(),
-        fleek::init_ops_and_esm(),
+        deno_io::deno_io::init_ops_and_esm(Some(Default::default())),
+        deno_fs::deno_fs::init_ops::<Permissions>(MaybeArc::new(InMemoryFs::default())),
+        deno_node::deno_node::init_ops_and_esm::<Permissions>(
+            None,
+            MaybeArc::new(InMemoryFs::default()),
+        ),
```

Comment:
same here reuse the same inmemoryfs for consenstency. Though not as crucial as it's never used for snapshotting